test/access/manager/AccessManager.predicate.js

```diff
@@ -1,27 +1,23 @@
 const { setStorageAt } = require('@nomicfoundation/hardhat-network-helpers');
-const {
-  time: { setNextBlockTimestamp },
-} = require('@nomicfoundation/hardhat-network-helpers');
-const { time } = require('@openzeppelin/test-helpers');
-const { EXECUTION_ID_STORAGE_SLOT, EXPIRATION, scheduleOperation } = require('../../helpers/access-manager');
+const { EXECUTION_ID_STORAGE_SLOT, EXPIRATION, prepareOperation } = require('../../helpers/access-manager');
 const { impersonate } = require('../../helpers/account');
-const { expectRevertCustomError } = require('../../helpers/customError');
+const { bigint: time } = require('../../helpers/time');
+const { keccak256, AbiCoder } = require('ethers');
+const { ethers } = require('hardhat');
 
 // ============ COMMON PREDICATES ============
 
 const LIKE_COMMON_IS_EXECUTING = {
   executing() {
     it('succeeds', async function () {
-      await web3.eth.sendTransaction({ to: this.target.address, data: this.calldata, from: this.caller });
+      await this.caller.sendTransaction({ to: this.target.target, data: this.calldata });
```

Comment:
`to` is `AddressLike`, so it should support providing the full object. If it works, it would be better.
```suggestion
      await this.caller.sendTransaction({ to: this.target, data: this.calldata });
```

I guess that applies to many places.